src/Mono.Android/Mono.Android.targets

```diff
@@ -352,5 +360,15 @@
         WorkingDirectory="$(MSBuildThisFileDirectory)"
     />
   </Target>
-  
+
+  <Target Name="_GenerateApiDocsDiff">
```

Comment:
What does this target do?  What's it for?  The commit message should mention it. :-)

Reply:
The description briefly mentions it:

> The api docs pipeline has been updated to allow these properties to be
set at queue time, and it will now upload a .diff file rather than
the entire docs folder.

Previously the API docs pipeline would upload the updated doc files, and now it will upload this diff file instead.